cdap-app-fabric/src/main/java/io/cdap/cdap/internal/capability/autoinstall/AutoInstall.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.internal.capability.autoinstall;
+
+import java.util.Objects;
+
+/**
+ * Class that has location of resources that need to be auto installed.
+ */
+public class AutoInstall {
+  private final String hubUrl;
```

Comment:
Use `URL` type to carry url.

Reply:
Also, it should be `hubURL`, not `hubUrl`

Reply:
As discussed on design doc, I have removed this and replaced it with an array of urls named "hubs".  The array is an array of strings because it's deserialized from json.